.github/workflows/typos.toml

```diff
@@ -0,0 +1,19 @@
+[files]
+extend-exclude = [
+    "*.svg",
+    "*.png",
```

Comment:
I think PNG files, as binary files, should be skipped by default, so we should be able to remove this?